### PR TITLE
removing gateway.watsonplatform.net

### PR DESCRIPTION
**Pull request by ljbennett62**
removed/updated service watson-platform-net url.  Can you test verify this pattern - as I tried to install the XCODE and don't seem to have enough space to install.

### CODE REVIEW

File: ReactNativeCodePattern/src/discovery.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import GLOBALS from '../Globals';
 
 export async function getDiscovery(keyword) {
   // use Discovery Query Language
-  const url_dql = `https://gateway.watsonplatform.net/discovery/api/v1/environments/system/collections/news-en/query?version=2018-08-01&aggregation=filter%28enriched_title.entities.type%3A%3ACompany%29.term%28enriched_title.entities.text%29.timeslice%28crawl_date%2C1day%29.term%28enriched_text.sentiment.document.label%29&filter=IBM&highlight=true&passages.count=5&query=${keyword}`;
+  const url_dql = `hhttps://api.us-south.discovery.watson.cloud.ibm.com/v1/environments/system/collections/news-en/query?version=2018-08-01&aggregation=filter%28enriched_title.entities.type%3A%3ACompany%29.term%28enriched_title.entities.text%29.timeslice%28crawl_date%2C1day%29.term%28enriched_text.sentiment.document.label%29&filter=IBM&highlight=true&passages.count=5&query=${keyword}`;
```
Comment (markstur) [inline ReactNativeCodePattern/src/discovery.js:5]: typo.  There is an extra h in hhttps

**Review by markstur — CHANGES_REQUESTED** [review]
There is a typo in the hhttps

**Review by markstur — COMMENTED** [review]
That's better.  When Rich tests this he should consider making it work in all regions instead of just us-south.  This looks like the correct first fix for removing the obsolete domain, but next I think it is too hard-coded (not a new problem though).

**Review by rhagarty — APPROVED** [review]
url change looks good

**Pull request merged**